build.gradle

```diff
@@ -42,18 +42,22 @@ task coverage(type: JacocoReport) {
 
 dependencies {
     String jUnitVersion = '5.4.0'
-    String javaFxVersion = '17.0.7'
+    String javaFxVersion = '17.0.11'
```

Comment:
@gongg21 Some of these changes are not strictly needed for the Java 17 migration. If you think they are still useful, you can send separate PRs for them, with the justification.